scripts/install.sh

```diff
@@ -159,16 +158,22 @@ celery worker -A ${INVENIO_WEB_INSTANCE}.celery -l INFO &
 # sphinxdoc-start-celery-worker-end
 
 # sphinxdoc-populate-with-demo-records-begin
-echo '{"title":"Invenio 3 Rocks", "recid": 1}'| \
-    python manage.py records create
+echo "from invenio_db import db; \
+from invenio_records import Record; \
+Record.create({'title': 'Invenio 3 rocks'}, \
+  id_='4b0714f6-9fe4-43d3-a08f-38c2b309afba'); \
+db.session.commit()" | python manage.py shell
 # sphinxdoc-populate-with-demo-records-end
```

Comment:
Yes I agree that it's somewhat cumbersome to use the CLI for ingesting records now. Not sure what is the best way forward though.